nix-ts-mode.el

```diff
@@ -173,15 +189,27 @@
 ;; Syntax map
 (defvar nix-ts-mode--syntax-table
   (let ((table (make-syntax-table)))
+    (modify-syntax-entry ?#  "<"      table)
+    (modify-syntax-entry ?\n ">"      table)
+    (modify-syntax-entry ?/  ". 124b" table)
+    (modify-syntax-entry ?*  ". 23"   table)
     table)
   "Syntax table for `nix-ts-mode'.")
 
+(defun nix-ts-mode--defun-name (node)
+  "Return the defun name of NODE.
+Return nil if there is no name or if NODE is not a defun node."
+  (pcase (treesit-node-type node)
+    ("binding"
+     (treesit-node-text
+      (treesit-node-child-by-field-name node "attrpath") t))))
+
 ;;;###autoload
 (define-derived-mode nix-ts-mode prog-mode "Nix"
   "Major mode for editing Nix expressions, powered by treesitter.
 
 \\{nix-ts-mode-map}"
-  :group 'nix
+  :group 'nix-ts
```

Comment:
```suggestion
```
Removing this entirely, since it will do the right thing automatically.

Reply:
Fixed, thanks!